sys/sys/elf_common.h

```diff
@@ -1337,6 +1345,7 @@ typedef struct {
 #define	R_RISCV_TLS_TPREL64	11
 
 /* Relocation types not used by the dynamic linker. */
+#define	R_RISCV_TLSDESC		12
```

Comment:
This is a dynamic relocation

Reply:
yeah, i interpreted this ambiguous existing comment as meaning "our dynamic linker doesn't currently use these" rather than "these could never be used by the dynamic linker"?

Reply:
Well, we don't support RV32, so the various 32-bit TLS relocations already fall into that category